zebra-rpc/src/methods/get_block_template_rpcs/types/subsidy.rs

```diff
@@ -58,15 +58,16 @@ pub struct FundingStream {
     ///
     /// The current Zcash funding streams only use transparent addresses,
     /// so Zebra doesn't support Sapling addresses in this RPC.
-    pub address: transparent::Address,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub address: Option<transparent::Address>,
```

Comment:
I opened #8702.

Reply:
zcashd will use [a separate "lockboxstreams" subobject](https://github.com/zcash/zcash/pull/6912/files#diff-ccc24453c13307f815879738d3bf00eec351417537fbf10dde1468180cacd2f1R938-R945) for that reason and as agreed on #8702.

Reply:
I think the changes for `getblocksubsidy` and `getblocktemplate` RPCs should be reversed if possible and addressed  them in the context of https://github.com/ZcashFoundation/zebra/issues/8702.

Reply:
> I think the changes for getblocksubsidy and getblocktemplate RPCs should be reversed if possible and addressed in the context of https://github.com/ZcashFoundation/zebra/issues/8702.

We need the changes to the `getblocktemplate` RPC method for it to produce valid NU6 blocks, and we need to make some change to `getblocksubsidy` since `funding_stream_address()` now returns an `Option`. I updated `getblocksubsidy` to exclude the lockbox funding stream from its response in https://github.com/ZcashFoundation/zebra/pull/8694/commits/14fbfb454f9fd58cb4c96b8ca9821048bb699a4f since we agreed we want to add information about the lockbox funding stream(s) in a different list.